agroal/README.md

```diff
@@ -0,0 +1,55 @@
+Integration with Agroal jdbc pooling library 
+===
+
+Author: Ondra Chaloupka<br/>
+Level: Intermediate<br/>
+Technologies: JTA, JDBC, Agroal
+
+
+What is it?
+---
+
+This examle demostrates usage of Agroal (https://agroal.github.io) jdbc pooling library
+in the standalone application while integrated with Narayana transaction manager.
+
+
+Prerequisities
+---
+
+JDK8 expected.
```

Comment:
expected or required?

Reply:
required for the quickstart. I will fix this.